models/preference.go

```diff
@@ -74,6 +74,12 @@ type LoadTestPreferences struct {
 	LoadGenerator      string `json:"gen,omitempty"`
 }
 
+// Parameters to updates Anonymous stats
+type PreferenceParams struct {
+	AnonymousUsageStats  bool `json:"anonymousUsageStats"`
+	AnonymousPerfResults bool `json:"anonymousPerfResults"`
+}
```

Comment:
These were missing before, huh?

Reply:
Nope, they are there in a model, But only these two were being used as parameters so I made a separate struct to add them as query parameters.

Reply:
@piyushsingariya You were not able to reuse `Preference` struct because it had additional fields which are not supposed to be a query parameter correct? And this struct is just for docs specifically correct?

Reply:
Yes, @kumarabd !